app/src/docs/components/DocsLayout/index.jsx

```diff
@@ -35,29 +198,55 @@ const DocsLayout = ({ nav = <DocsNav />, location, staticContext, ...props }) =>
                 footer
                 nav={nav}
             >
-                <Navigation
-                    responsive
-                />
+                {isSearching && (
+                    <Search
+                        nav={nav}
+                        toggleOverlay={toggleOverlay}
+                    />
+                )}
+                <MobileSearchButton
+                    type="button"
+                    onClick={() => toggleOverlay()}
+                >
+                    <SvgIcon name="search" />
+                </MobileSearchButton>
+                <MobileNav>
+                    {!isSearching && (
+                        <Navigation.Responsive />
+                    )}
+                </MobileNav>
                 <DocsContainer>
                     <div className={styles.grid}>
-                        <div>
-                            <Navigation />
-                        </div>
-                        <div className={styles.content}>
-                            {editFilePath && (
-                                <Button
-                                    className={styles.editButton}
-                                    tag="a"
-                                    href={`https://github.com/streamr-dev/streamr-platform/edit/development/app/src/docs/content/${editFilePath}`}
-                                    kind="secondary"
-                                    size="mini"
-                                    target="_blank"
-                                    rel="noopener noreferrer"
+                        <DesktopNav>
+                            <DesktopNavInner>
+                                <DesktopSearchButton
+                                    type="button"
+                                    onClick={() => toggleOverlay()}
                                 >
-                                    <SvgIcon name="github" className={styles.githubIcon} />
-                                    Edit on GitHub
-                                </Button>
-                            )}
+                                    <SvgIcon name="search" />
+                                    <SearchButtonText>Search</SearchButtonText>
+                                    <Key>⌘</Key>
```

Comment:
ctrl is also allowed, though this icon doesn't change... no room for anything else. cc @mattatgit